YSE_App/frb_observing.py

```diff
@@ -161,4 +162,64 @@ def ingest_obslog(obslog:pandas.DataFrame, user, override:bool=False):
         frb_status.set_status(transient)
 
     return 200, "All good"
+    
+def ingest_z(z_tbl:pandas.DataFrame, user):
+    """ Ingest a table of redshifts into FFFF-PZ
+
+    This updates the transient status 
+
+    Args:
+        z_tbl (pandas.DataFrame): table of redshifts
+            TNS (str) -- TNS of the FRB that has this galaxy as its preferred host
+            Galaxy (str) -- JNAME *matching* that in FFFF-PZ
+            Resource (str) -- Name of the FRB Followup Resource
+            Redshift (float) -- Redshift of the galaxy (float)
+            Quality (int) -- Quality of the redshift (int)
+        user (_type_): _description_
+
+    Returns:
+        tuple: (status, message) (int,str) 
+    """
+
+    # TODO
+    # Scrub previous entries with the named resource?
+    
+    # Loop on rows
+    for _, row in z_tbl.iterrows():
```

Comment:
Is this iterating through all the rows? If so, that explains the error I was getting. Because it works when you have only new FRBs in the table